accounting/src/delta/delta_data_collection/mod.rs

```diff
@@ -14,52 +14,94 @@
 // limitations under the License.
 
 pub mod undo;
+use self::undo::*;
 
 use std::collections::BTreeMap;
 
 use serialization::{Decode, Encode};
+use utils::ensure;
```

Comment:
I really don't think this is necessary... import the whole utils module just for this thing... I don't really care for it. I'd rather have one less dependency unless we must.